pkg/driver/mount.go

```diff
@@ -362,7 +362,9 @@ func (m *NodeMounter) MakeFile(pathname string) error {
 			return err
 		}
 	}
-	defer f.Close()
+	defer func() {
+		f.Close()
+	}()
```

Comment:
As this is the last line in the function the easier way to fix this would be to just remove the defer.
```suggestion
	f.Close()
```